src/utils.ts

```diff
@@ -1,4 +1,5 @@
 import { firestore, GeoFirestoreObj, QueryCriteria } from './interfaces';
+import { triggerAsyncId } from 'async_hooks';
```

Comment:
What is this used for?

Reply:
Thank you! Removed in https://github.com/geofirestore/geofirestore-js/pull/53/commits/324aa9604d6e9419711eac77d1014706eafe3791

Probably my (overly) helpful editor tough this project needed this.